_data/kautukkundan/kautukkundan.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Kautuk Kundan
+institution: BVCOE, New Delhi
+profile_pic: kautukkundan.jpg
+quote: "To achieve something you never had, do something you have never done"
```

Comment:
Consider avoiding quotes. From README:
> quote: YOUR-SENIOR-QUOTE # no longer than 100 characters, avoid using quotes(") to guarantee the format remains the same.

Reply:
Didn't catch that. I aplogise for my review mistake. Thanks a lot.

Reply:
@ayan-b Thank you for pointing that out! corrected